vllm/config.py

```diff
@@ -106,6 +106,7 @@ def __init__(
         disable_sliding_window: bool = False,
         skip_tokenizer_init: bool = False,
         served_model_name: Optional[Union[str, List[str]]] = None,
+        multimodal_config: Optional["VisionLanguageConfig"] = None,
```

Comment:
Related to this, would you be open to renaming `VisionLanguageConfig` to `MultimodalConfig`? I tried adding in a separate `AudioLanguageConfig` for audio modalities but ended up having to change dozens of call sites, and I tend to think having a single multimodal config that can be used for any modality would be the simplest way forward.

Reply:
Yes, that's exactly my plan. However, that is out of scope for this PR.

Reply:
Oh, agreed regarding the revamp, I was simply suggesting the name change here.